packages/loopring_v3/contracts/amm/LoopringAmmPool.sol

```diff
@@ -98,9 +98,17 @@ contract LoopringAmmPool is
         external
         nonReentrant
     {
+        require(state.accountID == 0 || msg.sender == address(controller), "UNAUTHORIZED");
```

Comment:
even when state.accountID == 0, should we still only allow **only** the controller can set up the pool?

Reply:
Currently `setupPool` is called directly after the proxy contract is created in `LoopringAmmFactory`, so that would have to be changed. But I think it should be fine, I don't think there's any risk with allowing anyone to call setupPool because it's called in the same Ethereum tx as the proxy is created.